processor/k8sattributesprocessor/README.md

```diff
@@ -75,9 +75,11 @@ Additional container level attributes can be extracted provided that certain res
    - k8s.container.name
    - container.image.name
    - container.image.tag
+   - container.image.repo_digests (if provided by the container runtime)
```

Comment:
I wonder if this could be a bit more explanatory 🤔

Reply:
Maybe "if k8s CRI populates repository + digest".
But then we need to explain what CRI means to people who don't really know/care about that.
Maybe just a link to the explanation in the semantic conventions? https://github.com/open-telemetry/semantic-conventions/blob/f79fe6b61438bfa0804df9c254c402fd0d71f42a/model/registry/container.yaml#L60-L71

Reply:
Would adding both work here? I think it's better to explicitly surface this technical detail, even if that might be unknown to some users (they can always learn about it) rather than hiding it.

Reply:
Added link to semantic conventions registry and enhanced comment about how repo_digests field will get populated.